spec/fixtures/fixed_fields/marc_with_bad_008.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<collection>
+    <record>
+        <leader>01104nma a2200289 i 4500</leader>
+        <controlfield tag="001">99127156537506421</controlfield>
+        <controlfield tag="008">230519s1996    njua       k  000 0 eng d</controlfield>
```

Comment:
I would like to have the 008 have a value that can be fixed. In the `fix_comp_008` method, that means that it has to be one of the following positions of the field:

audience (position 22 of `u` or `v` becomes `j`)
government publication indicator (position 28 of 'n' becomes 'o')